dex-operator-image/dex-operator-image.kiwi.ini

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: _PRODUCT_/dex-operator:%%LONG_VERSION%% _PRODUCT_/dex-operator:%%LONG_VERSION%%-<RELEASE> -->
+
+<image schemaversion="6.5" name="_PRODUCT_-dex-operator">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>Dex operator running on a _DISTRO_ container guest</specification>
+  </description>
+  <preferences>
+    <type
```

Comment:
I think it's a bit unusual to break xml tags like this. Ideally it should be

```
<type image="docker" derived_from="obsrepositories:/_BASEIMAGE_">

Reply:
We don't want to have too long lines